Exiled.API/Features/Effect.cs

```diff
@@ -15,7 +15,7 @@ namespace Exiled.API.Features
     using Exiled.API.Extensions;
 
     /// <summary>
-    /// Useful class to save effect configs in a cleaner way.
+    /// Useful class to save effects in configs in a cleaner way.
```

Comment:
This doesn't sound right to me. Maybe something like "Useful class to save effect-related configs cleanly."?

Reply:
That sounds pretty good, I have changed it to match that